docs/install.rst

```diff
@@ -111,6 +179,75 @@ When using conda, ffmpeg can be installed as follows::
     conda install -c conda-forge ffmpeg
 
 
+Conda environments
+------------------
+.. _PyCharm: https://www.jetbrains.com/pycharm/download/#section=windows
+.. _PyCharm documentation: https://www.jetbrains.com/help/pycharm/conda-support-creating-conda-virtual-environment.html
+.. _Jupyter: https://jupyter.org/
+.. _uninstall instructions: https://docs.anaconda.com/anaconda/install/uninstall/
+
+If you have installed Pylake according to the Installation Instructions for Anaconda, then you should now have a separate environment for your Pylake work.
+You may be wondering why we needed to create a new environment for Pylake, and what an environment is.
+
+When using Python, you will quickly find that several packages that you can install depend on each other.
+For example, when installing Pylake, you could see that the installation of Pylake required numerous other packages to be installed as well.
+Most Python packages are continuously updated by their authors. Sometimes, these authors decide that certain existing functionalities need to be changed.
+This means that not all of them will be completely backwards compatible.
+Therefore, it can be challenging to find a suitable collection of packages that fulfills the requirements of the package you are trying to install, while ensuring that all packages are still compatible.
+
+One pragmatic solution to this is to maintain separate Python environments for different projects.
+This means the solver doesn't have a large number of dependencies to resolve.
```

Comment:
```suggestion
This means that you create independent "copies" of Python and its installed packages, so that the different projects you're working on don't interfere with each other.
```
("solver" is not something you've defined; trying to rephase more in layman terms)

Reply:
Ah yes, much better. Amended! 👍